packages/app/src/composables/useSpecFilter.ts

```diff
@@ -25,7 +25,6 @@ export function useSpecFilter (savedFilter?: string) {
 
   function setSpecFilter (specFilter: string) {
     if (specStore.specFilter !== specFilter) {
-      specStore.setSpecFilter(specFilter)
```

Comment:
Unused, this was not doing anything but saving the specFilter in the store for no reason.